tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb

```diff
@@ -6,21 +6,23 @@ class UploadSymbolsToBugsnagAction < Action
         File.join(File.dirname(__FILE__), '..', '..', '..', '..', '..', 'bugsnag-dsym-upload'))
 
       def self.run(params)
-        options = {}
-        options = options_from_info_plist(params[:config_file]) if params[:config_file]
-
-        if (params[:config_file] == default_info_plist_path || options[:apiKey].nil? || options[:apiKey].empty?)
-          # Load custom API key if config file has not been overridden or the API key couldn't be found in config files
-          options[:apiKey] = params[:api_key] unless params[:api_key].nil?
-        else
-          # Print which file is populating version and API key information since the value has been
-          # overridden
-          UI.message("Loading API key from #{params[:config_file]}")
+        # If we have not explicitly set an API key through env, or parameter
+        # input in Fastfile, find an API key in the Info.plist in config_file param
+        if params[:config_file] && params[:api_key] == nil
+          UI.message("Using the API Key from #{params[:config_file]}")
+          params[:api_key] = options_from_info_plist(params[:config_file])[:apiKey]
+        end
+
+        if params[:verbose]
+          UI.message("Uploading dSYMs to Bugsnag with the following parameters:")
```

Comment:
Why do we not use `UI.verbose` here?

Reply:
This is a parameter that was defined in a previous iteration, we can move to use `UI.verbose` and adjust the documentation accordingly? This also brings it inline with the build reporter which will use `UI.verbose` too.

Reply:
Well I'm assuming that fastlane has a verbose mode internally that you can switch on so it seems like we should hook into that rather than making our own.

Reply:
Done: https://github.com/bugsnag/bugsnag-dsym-upload/pull/42/commits/a910a01c18ea50c067d3f42fa459484deb5edea0